app/design/adminhtml/default/default/template/sales/order/view/info.phtml

```diff
@@ -28,7 +28,7 @@ $orderStoreDate = $this->formatDate($_order->getCreatedAtStoreDate(), 'medium',
         endif; ?>
         <div class="entry-edit-head">
         <?php if ($this->getNoUseOrderLink()): ?>
-            <h4 class="icon-head head-account"><?php echo Mage::helper('sales')->__('Order # %s', $this->escapeHtml($_order->getRealOrderId())) ?> (<?php echo $_email ?>)</h4>
+            <h4 class="icon-head head-account copy-text" data-text="<?= $_order->getRealOrderId() ?>" data-copy-icon='{"cursor":"pointer","margin":"8px 0 0 5px","height":"16px","width":"16px"}'><?php echo Mage::helper('sales')->__('Order # %s', $this->escapeHtml($_order->getRealOrderId())) ?> (<?php echo $_email ?>)</h4>
```

Comment:
do we have to have all of this data in the data-copy-icon? can we have like a couple of classes or something easier?  if we have all the css in a json, at this point it's better to have the span already.

Reply:
I don't understand the need to specify a CSS for each element in `data-copy-icon`

Reply:
why not include all in css like this?
![icon-copy](https://github.com/OpenMage/magento-lts/assets/5071467/dfa0993e-ea29-4278-adb0-0e4bd0854f4a)

![demo](https://github.com/OpenMage/magento-lts/assets/5071467/5aa231ab-713d-4025-aec5-9a3704883aaa)


keep the same icon size is better for me, but we can alway stylish in css like:

```
.entry-edit-head .icon-copy {
width:16px
height:16px;
}
```

Reply:
My idea is to allow users to add the copy icon functionality to any element in the adminhtml without adding an element near the target element.

Reply:
I understand that but if you've to have all the css in a json data-attribute, then render the actual element via js... to me it's a no.

Reply:
minimum the width/height and the cursor should have defaults

Reply:
ok, so create span via js or add placeholder in template? which is better?

Reply:
if the css in data-attribute was optional then I'd be ok with the current implementation. width/height to 16 by default (or 12, what makes more sense), and the cursor too. didn't check about the margins. then override the margins if necessary.

my opinion is partially positive because it's backend, if it was frontend this approach could cause content layout shift and should be avoided.

-----------------

actually
- if data-text was renamed to data-copy-text then would probably be no need for the class='copy-text' in order to identify the elements that need the copy icon next to it
- also instead of width/height in the data-copy-icon I'd just have a size, they have to be square, so
`<h4 class="icon-head head-account copy-text" data-text="<?= $_order->getRealOrderId() ?>" data-copy-icon='{"cursor":"pointer","margin":"8px 0 0 5px","height":"16px","width":"16px"}'>` could be `<h4 class="icon-head head-account" data-copy-text="<?= $_order->getRealOrderId() ?>" data-copy-icon='{"margin":"8px 0 0 5px","size":"16px"}'>` and only if the size if overriding the default size

Reply:
i vote for remove all `data-copy-icon` and stylish it via css

Reply:
true, since you've to modify the phtml anyway... it not a super big benefit to have the data-attribute instead of just a <span